src/robusta/core/sinks/robusta/dal/model_conversion.py

```diff
@@ -97,6 +97,8 @@ def to_evidence_json(
                 structured_data.append({"type": "list", "data": block.items})
             elif isinstance(block, PrometheusBlock):
                 structured_data.append({"type": "prometheus", "data": block.data.json(), "metadata": block.metadata})
+            elif isinstance(block, EventsBlock):
```

Comment:
will this prevent us from creating findings from any of the events actions since they all get the events from get_resource_events_table? can you verify we get events in other findings in the platform still?

Reply:
This will only prevent the creation of events evidence. 
Finding will be created and extra code from the UI will be added to fetch the events. 

There is an extra annotation to mute the finding but it has to be added specifically   

I will run some extra integration tests once the UI is ready as well

Reply:
I think @Avi-Robusta is right.
When the UI calls the action, finding is not created in the DB.
It's called with `no sinks`. So the data is just returned by the websocket response